plugins/inputs/deprecations.go

```diff
@@ -0,0 +1,42 @@
+package inputs
+
+import "github.com/influxdata/telegraf"
+
+var Deprecations = map[string]telegraf.DeprecationInfo{
+	"cassandra": {
+		Since:  "1.7",
+		Notice: "use 'inputs.jolokia2' with the 'cassandra.conf' example configuration instead",
+	},
+	"http_listener_v2": {
+		Since:  "1.9",
+		Notice: "has been renamed to 'influxdb_listener', use 'inputs.influxdb_listener' or 'inputs.influxdb_listener_v2' instead",
+	},
+	"httpjson": {
+		Since:  "1.6",
+		Notice: "use 'inputs.http' instead",
+	},
+	"jolokia": {
+		Since:  "1.5",
+		Notice: "use 'inputs.jolokia2' instead",
+	},
+	"kafka_consumer_legacy": {
+		Since:  "1.20",
+		Notice: "use 'inputs.kafka_consumer' instead, NOTE: 'kafka_consumer' only supports Kafka v0.8+",
+	},
+	"logparser": {
+		Since:  "1.15",
+		Notice: "use 'inputs.tail' with 'grok' data format instead",
+	},
+	"snmp_legacy": {
+		Since:  "1.20",
```

Comment:
Isn't this already deprecated for way longer?

Reply:
I put the current version for plugins where I could not find that number stated somewhere. For `snmp_legacy`, `riemann_legacy` and I think `kafka_consumer_legacy` are the once where I could not find a statement about the deprecation version.

Reply:
Let me know which version it is and I can fix it.

Reply:
It would need a deep search in git history indeed, will try to find it..

I think it is done like this when new snmp input is created, old plugin was renamed to _legacy and the new one took over its name.

Reply:
Found it... 1.0 8-)